frontend/ui/detailview.jsx

```diff
@@ -420,6 +428,25 @@ class DetailViewComponent extends React.Component {
   }
 
   render() {
+    let crashStatsLink;
+    if (this.state.measure in CRASH_STATS_MAPPING) {
+      const { processType, extraParams } = CRASH_STATS_MAPPING[this.state.measure];
+      const queryParams = stringify({
```

Comment:
stringify looks cool, I did not know about it. There are some other places in this view where I think it could be used. If you wanted to do a followup doing that, that would be cool. :)

Reply:
Filed as #168